src/patcher/java/net/minecraftforge/gradle/patcher/PatcherPlugin.java

```diff
@@ -586,12 +590,6 @@ public void apply(@Nonnull Project project) {
             if (userdevConfig.get().getSource() == null) {
                 userdevConfig.get().setSource(project.getGroup().toString() + ':' + sourcesJar.get().getBaseName() + ':' + project.getVersion() + ':' + sourcesJar.get().getClassifier() + '@' + sourcesJar.get().getExtension());
             }
-            if (!"a/".contentEquals(genConfig.get().getOriginalPrefix())) {
-                userdevConfig.get().setPatchesOriginalPrefix(genConfig.get().getOriginalPrefix());
-            }
-            if (!"b/".contentEquals(genConfig.get().getModifiedPrefix())) {
-                userdevConfig.get().setPatchesModifiedPrefix(genConfig.get().getModifiedPrefix());
-            }
```

Comment:
We need these prefixes for patch gen for old patches so it doesn't diff the crap out of existing ForgeDev workspaces.